templates/organizational/organizational-arms-en.html

```diff
@@ -13,14 +13,14 @@
 	"share": "true"
 }
 ---
-<h1 property="name" id="wb-cont">{{{title}}}</h1>
+<h1 property="name" id="wb-cont">{{ page.title }}</h1>
 <div class="row">
 	<div class="col-sm-7 pull-left">
 		<p class="gc-byline">We are a federal institution that operates at arm’s length from the Government of Canada.</p>
 		<p>[Institution name] Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor. Cras sagittis massa sed libero eleifend, ac iaculis urna tempor.</p>
 	</div>
 	<div class="col-sm-5 col-xs-12 pull-right">
-		{{>follow}}
+		{% include templates/follow.html %}
```

Comment:
"Follow us", "most requested" are reused components which should be defined in their own folder.

They are documented separately in the design system
* https://design.canada.ca/common-design-patterns/social-media-channels.html
* https://design.canada.ca/common-design-patterns/most-requested.html

So the proper fix will be to create 2 new components:
*  "/components/followus/includes/"
*  "/components/most-requested/includes/"

or create sample here:
*  "/components/followus/samples/"
*  "/components/most-requested/samples/"

---

Maybe we can consider to use "include_relative" instead of "include" for this kind of situation where template are using other components or using a component "samples".